assets/app/view/game/corporation.rb

```diff
@@ -308,23 +310,22 @@ def render_revenue_history
 
       def render_abilities(abilities)
         attribute_lines = []
+        props = { style: { display: 'inline-block' } }
 
         abilities.each do |ability|
-          attribute_lines << h('div.name.nowrap', ability.name)
-          attribute_lines << h('div.right', ability.display_value || '')
+          attribute_lines << h('div.nowrap', props, ability.name)
         end
 
         table_props = {
           style: {
-            padding: '0 0.5rem',
-            grid: 'auto / 3fr 2fr',
-            gap: '0 0.2rem',
+            padding: '0.5rem',
+            grid: 'auto',
```

Comment:
grid & gap are obsolete if you’re just using lines

Reply:
Fixed

Reply:
Removed now. And put in centralized as you had in your commit.